.gitlab/macrobenchmarks.yml

```diff
@@ -0,0 +1,221 @@
+variables:
+  BENCHMARKS_CI_IMAGE: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/benchmarking-platform:go-go-prof-app
+
+.benchmarks:
+  stage: macrobenchmarks
+  tags: ["runner:apm-k8s-same-cpu"]
+  timeout: 1h
+  rules:
+    - if: $CI_PIPELINE_SOURCE == "schedule"
```

Comment:
Please use the same template as in https://github.com/DataDog/dd-trace-rb/blob/master/.gitlab/benchmarks.yml#L13-L16